packages/amplication-client/src/Workspaces/WorkspaceHeader/WorkspaceHeader.tsx

```diff
@@ -99,6 +100,13 @@ const WorkspaceHeader: React.FC = () => {
     }
   }, []);
 
+  const onBuildNotificationClick = useCallback(
+    (templateIdentifier: string, type: ButtonTypeEnum, message: IMessage) => {
+      console.log(templateIdentifier, type, message);
```

Comment:
1. Isn't there an implementation missing here on the onClick? Or is this just for your debugging? If the latter - perhaps remove this. If the former, perhaps you need to commit something you haven't.
2. Do you want to leave that on the console.log level (that is only visible on the dev tools) or send this log upstream for us?

Reply:
I don't have exactly the logic under the button click, so I just added the wrap function. waiting for specific logic from product